ssh/tailssh/tailssh.go

```diff
@@ -357,6 +361,24 @@ func (srv *server) handleAcceptedSSH(ctx context.Context, s ssh.Session, ci *ssh
 	// See https://github.com/tailscale/tailscale/issues/4146
 	s.DisablePTYEmulation()
 
+	var rec *recording // or nil if disabled
+	var err error
+	ptyReq, _, isPtyReq := s.Pty()
+	if recordSSH && isPtyReq { // for now only record pty sessions
+		env := envMap(s.Environ())
```

Comment:
Should this be cmd.Env?

Reply:
Probably, yeah, but it requires rearranging a bunch of stuff. We want to get this env before we start the process.

Another reason to make a new type.